docs/DeveloperGuide.md

```diff
@@ -8,22 +8,72 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Update patient information
+Step 1:Patient inform the nurse of their change in personal information. Nurse will open up the Duke system and request to submit the change. Duke will call the Parser#etPatientInfo() to update the patient information. 
+
+Step 2:
+The parser will call the database storing patient information to update the change.
+
+The flow is presented in the sequence diagram (will be included later after pull request is permitted):
+
+
+### Exit program
+Step 1: User type "bye" as input to exit the program. IHospital will call Parser#getCommand()
+to return the user command received.
+
+Step 2: After checking the user command is "bye", IHospital will
+call Ui#sayGoodbye to print Goodbye message
+
+The following sequence diagram shows how the exit operation works:
+
+![](ByeCommand.png)
```

Comment:
![image](https://user-images.githubusercontent.com/15943697/160964103-33c79911-faff-4220-abd8-83c9c491f16b.png)
For the sequence diagram, consider naming it as fullCommand:datatype, assuming if your return from parser is a string, you could put fullCommand:String instead. 

![image](https://user-images.githubusercontent.com/15943697/160964123-5573453d-6f50-4424-b63d-2ddb65f61bbe.png)
Similarly for your last 2 return arrows containing the Goodbye message you could consider putting it as "message:String" instead.

![image](https://user-images.githubusercontent.com/15943697/160964138-39705e4a-1eb9-4e97-ae97-b1fd9dff0645.png)
For the arguments in the methods for saveDoctorList(), savePatientList(), saveApppointmentList() consider putting the data type as well. For example, "doctors:ArrayList<Doctors>" if your data type is `ArrayList<Doctors>`.

![image](https://user-images.githubusercontent.com/15943697/160964163-53b4c23f-d069-4505-8868-c00c0aa2a550.png)
All of the object calls are missing the activation bar, consider adding them.

![image](https://user-images.githubusercontent.com/15943697/160964257-64c88729-84bf-432f-90d7-ce06e4860f51.png)
Consider removing the extra boxes that appear at the bottom, I think PlantUML can remove it, if not so you may have to remove it manually.